### DISCUSSION TITLE
Feature Request: Set up Diagnostic Logging for Azure ML workspace

### DISCUSSION THREAD

**Discussion by denniseik** [Azure/data-product-analytics#80] [Q&A] [answered]
As part of the provisioning of the Azure ML workspace, we should set up diagnostic logging into a central log analytics workspace.

**Comment by marvinbuss** [comment, marked as answer]
Our recommendation for customers is to leverage Policy to do this at scale. Otherwise, customers would have to do this for each and every workspace. [This policy](https://github.com/Azure/data-management-zone/blob/main/infra/Policies/PolicyDefinitions/DiagnosticSettings/params.machineLearning.json) allows to do that at scale. This policy needs to be setup once and can then be used to capture logs of all workspaces within a tenant or a specific scope within the tenant. That is the reason why we are not setting this up manually.
